cocos/core/renderer/scene/directional-light.ts

```diff
@@ -35,7 +35,7 @@ const _v3 = new Vec3();
 
 export class DirectionalLight extends Light {
     protected _dir: Vec3 = new Vec3(1.0, -1.0, -1.0);
-    protected _illuminance: number = Ambient.SUN_ILLUM;
+    protected _illuminanceHDR: number = Ambient.SUN_ILLUM;
```

Comment:
This is internal property, we can refactor directly as long as we change everywhere it's been used